packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1431,7 +1526,14 @@ export default mixins(
 					return Promise.resolve();
 				}
 
-				if (this.$route.name === 'ExecutionById') {
+				if (this.blankRedirect) {
+					this.blankRedirect = false;
```

Comment:
avoid running new page code when moving from `workflows/template/:id` to `/workflow` new page